components/preparing_workspace/preparing_workspace.scss

```diff
@@ -82,3 +82,10 @@
     // centering
     margin: 0 auto;
 }
+
+@media screen and (max-width: 768px) {
+    #root,
+    body.app__body #root{
+        overflow-y: visible;
+    }
```

Comment:
The change seems to address the main issue of not being able to scroll down and access the action buttons at the bottom, but still looks odd at the top. If this is enough for the scope of the ticket then I'm ok with it:

![Feb-01-2023 13-40-24](https://user-images.githubusercontent.com/79058848/216134326-40dcf49d-0242-4c5d-9c3b-cca94ba91d65.gif)


Tested another approach that could help reduce the weird look at the top, but the original structure of the section doesn't help much. Again, only if really required for this ticket:

![Feb-01-2023 13-41-54](https://user-images.githubusercontent.com/79058848/216135008-d335adb8-a8bf-4574-bbac-9d010f12fbec.gif)

Reply:
Hello @julmondragon 
Thanks a lot for pointing the linter, and the suggestion for the padding top. 
the top we planned to have a new text, so it would be better for small screens, but this fix you posted would already look better for now
both changes are applied 👍